tools/mpremote/mpremote/transport.py

```diff
@@ -25,6 +25,7 @@
 # THE SOFTWARE.
 
 import ast, hashlib, os, sys
+import errno
```

Comment:
Please put this import on the line above to match style in this file.

Reply:
ok